frost-core/src/frost.rs

```diff
@@ -223,12 +275,11 @@ where
     ///
     /// [`compute_group_commitment`]: https://www.ietf.org/archive/id/draft-irtf-cfrg-frost-03.html#section-4.4
     fn try_from(signing_package: &SigningPackage<C>) -> Result<GroupCommitment<C>, &'static str> {
-        let rho: Rho<C> = signing_package.into();
+        let binding_factor_list: BindingFactorList<C> = signing_package.into();
```

Comment:
Should we use `rho_list`? Should we rename `Rho` to `BindingFactor`? Or is it good as is?

Reply:
We can leave for now, but the spec is /almost/ scrubbed of rho except internal to one subroutine, we should update the spec and rename our `Rho` in one go: https://github.com/ZcashFoundation/frost/issues/143